test/sanity/validate-modules/ignore.txt

```diff
@@ -786,8 +786,6 @@ lib/ansible/modules/network/ios/ios_logging.py E324
 lib/ansible/modules/network/ios/ios_ping.py E324
 lib/ansible/modules/network/ios/ios_vlan.py E325
 lib/ansible/modules/network/ios/ios_vrf.py E325
-lib/ansible/modules/network/iosxr/iosxr_command.py E322
-lib/ansible/modules/network/iosxr/iosxr_logging.py E324
```

Comment:
Restore these lines since these files were not modified in this PR.

Reply:
Ha, what a stupid mistake I made. Sorry about that.